gather_istio.sh

```diff
@@ -204,18 +205,23 @@ function main() {
   inspect clusterserviceversion "${operatorNamespace}"
 
   for cp in ${controlPlanes}; do
-      if [[ -z $(oc get smcp -n "${cp}" -oname) ]]; then
+      smcpName="$(oc get smcp -n "${cp}" -o jsonpath='{.items[*].metadata.name}')"
+      if [[ -z "$smcpName" ]]; then
         echo "ERROR: namespace ${cp} does not contain a ServiceMeshControlPlane object"
         exit 1
       fi
 
+      echo
       echo "Processing control plane namespace: ${cp}"
 
       inspect "ns/${cp}"
       for crd in ${crds}; do
         inspect "${crd}" "${cp}"
       done
 
+      inspect "mutatingwebhookconfiguration/istiod-${smcpName}-${cp}"
+      inspect "validatingwebhookconfiguration/istio-validator-${smcpName}-${cp}"
+
```

Comment:
Would those names vary depending on the istio version? Or could we expect different sets of resources based on the version?

Reply:
These are currently the same across all v2.x versions.